fitbit-react/src/Util/sort.js

```diff
@@ -0,0 +1,11 @@
+const sortByHighToLow = (products) => {
+  products.sort(
```

Comment:
Suggestion: use `productList` instead of `products`, because it's sorting and returning a product list not a single product